pkg/genericapiserver/config.go

```diff
@@ -71,6 +71,7 @@ type Config struct {
 	// allow downstream consumers to disable the index route
 	EnableIndex           bool
 	EnableProfiling       bool
+	EnableVersion         bool
```

Comment:
> Is /version/openshift used as a replacement of kube's /version? What will a kubectl used against an origin cluster see and use?

When you change what `/version` returns, `--match-server-version` on `kubectl` goes crazy.  I'd be happy to eliminate that flag (we haven't found much use for it downstream), but its not a generic endpoint.

Even if it were, this should change to specify what you want served.

I'll settle for this to get the rest of the pull, but `Version` as it is today is not generic